explorer/app/apis/anvil-catalog/common/transformers.ts

```diff
@@ -0,0 +1,21 @@
+import { AnvilSourceItem } from "../../anvil/common/entities";
+
+export const getStudyName = (source: AnvilSourceItem): string => source?.name;
```

Comment:
I think we can remove the optional chaining (`source?`) as the `source` param isn't optional.

Would you mind adding nullish coalescing (returning "") to each function here in the transformer so the transformer functions always return a string value (and not undefined or null).